controllers/nginx/Makefile

```diff
@@ -24,91 +20,20 @@ PKG=k8s.io/ingress/controllers/nginx
 
 ARCH ?= $(shell go env GOARCH)
 GOARCH = ${ARCH}
-DUMB_ARCH = ${ARCH}
```

Comment:
I replicated the build pipeline we were using in shopify-docker-images. I'll try use the default makefile and pipa constraints